src/ProjectAudioManager.cpp

```diff
@@ -824,6 +824,9 @@ bool ProjectAudioManager::DoRecord(AudacityProject &project,
             // remember the original in preroll tracks, before making the
             // pending replacement.
             const auto shared = wt->SharedPointer<WaveTrack>();
+            // playbackSequences contains only leaders; prerollSequences should
+            // be a subset of it.  Non-leader might not be found, but that is
+            // all right.
             bool prerollTrack =
```

Comment:
Can probably be `const`?

Reply:
Small advantage for readability, none for performance